atlas/packages/app/tsconfig.json

```diff
@@ -1,26 +1,29 @@
 {
-	"extends": "../../tsconfig.json",
-
-	"compilerOptions": {
-		"baseUrl": "./",
-		"declaration": true,
-		"declarationDir": "dist",
-		"lib": ["DOM", "ES2019"],
-		"paths": {
-			"components": ["src/components"],
-			"shared": ["src/shared"],
-			"shared/*": ["src/shared/*"],
-			"store": ["src/store"],
-			"theme": ["src/theme"],
-			"views": ["src/views"]
-		}
-	},
-
-	"exclude": [
-		"node_modules", "dist", "src/shared/stories"
-	],
-
-	"include": [
-		"src/shared/icons/types.d.ts"
-	]
+  "extends": "./tsconfig.paths.json",
```

Comment:
It is a bit of an overkill to have another file just for the paths imho, also looks like some options here are shared by the workspace root `tsconfig.json` , so those could be probably be moved all in the workspace root and inherited here.

Reply:
This isn't really done for convenience, rather as a "hack". CRA doesn't allow some options in `tsconfig.json` (look at Caveats in my comment above). If it detected some options it doesn't allow, it will try to delete them. Keeping them in a separate config allows us to alleviate that, so the paths aren't removed. Once again, if we decide to eject we can tweak that and get rid of this.

Regarding moving some options to the root, I will look into that

Reply:
> Regarding moving some options to the root, I will look into that

Maybe let's do that during rebasing on common linting